symbolic/src/Data/Macaw/Symbolic/CrucGen.hs

```diff
@@ -1755,6 +1756,22 @@ addSwitch blockLabelMap idx possibleAddrs = do
   mapM_ addStmt stmts
   addTermStmt termStmt
 
+
+addRegUpdateForBlock :: (OrdF (M.ArchReg arch)) => MacawSymbolicArchFunctions arch -> M.ArchSegmentOff arch -> CrucGen arch ids s ()
```

Comment:
I know that this is a non-exported function but might be worth a Haddock explaining *why* we want to do this.